source/frontend/public/mockServiceWorker.js

```diff
@@ -8,121 +8,121 @@
  * - Please do NOT serve this file on production.
  */
 
-const INTEGRITY_CHECKSUM = '3d6b9f06410d179a7f7404d4bf4c3c70';
-const activeClientIds = new Set();
+const INTEGRITY_CHECKSUM = '3d6b9f06410d179a7f7404d4bf4c3c70'
+const activeClientIds = new Set()
```

Comment:
This file gets re-generated whenever we do `npm install` - I personally ignore the changes but I'm okay changing it to have semicolons

Reply:
reverted changes to this file